website/src/components/TimeIcon.tsx

```diff
@@ -5,7 +5,7 @@ import { HourglassDynamic } from '@edgeandnode/gds/icons'
 
 import { useI18n } from '@/i18n'
 
-export interface TimeIconProps extends ComponentProps<'div'> {
+interface TimeIconProps extends ComponentProps<'div'> {
```

Comment:
ah. nice catch it was probably my mistake 😅

Reply:
Not really a mistake, haha, just removed it for consistency.